clients/ops/privacy-center/constants/index.ts

```diff
@@ -3,7 +3,9 @@
 
 import config from "../config/config.json";
 
+// Compute the host URL for the server, while being backwards compatible with
+// the previous "fidesops_host_***" configuration
 export const hostUrl =
   process.env.NODE_ENV === "development"
-    ? config.fidesops_host_development
-    : config.fidesops_host_production;
\ No newline at end of file
+    ? (config.server_url_development || (config as any).fidesops_host_development)
```

Comment:
Will this backwards compatibility check ever be removed?

Reply:
Fair question. I'll open an issue for it - I'm looking for ways to minimize migration pain where possible 👍

Reply:
Opened this: https://github.com/ethyca/fides/issues/1264

Reply:
That makes sense. Maybe some kind of deprecation warning would be appropriate here. Definitely a follow up ticket issue though.